### PR TITLE
fixed add button on wallet add network

### PR DESCRIPTION
**Pull request by qamarngr**
Fixes brave/brave-browser#22220
Fixes brave/brave-browser#21056

Resolves 
brave/brave-browser#22220
brave/brave-browser#21056
## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

**Review by SergeyZhukovsky — APPROVED** [review]
++

**Comment by Pavneet-Sing** [comment]
- [Either use “Resolves” or “Fixes”](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
- Could you please add a picture or gif of the final output

File: android/java/res/layout/brave_wallet_add_network.xml [hunk 1]
```diff
@@ -219,20 +219,21 @@
...
             android:gravity="center"
             android:layout_weight = "1"
             android:background="@drawable/crypto_wallet_blue_button"
             android:text="@string/brave_wallet_add_network_add"
             android:textAllCaps="false"
             android:textSize="16sp"
             android:layout_marginTop="10dp"
-            android:layout_marginStart="24dp"
+            android:layout_marginStart="12dp"
             android:layout_marginEnd="12dp"
```
Comment (Pavneet-Sing) [inline android/java/res/layout/brave_wallet_add_network.xml:231]: nit: can replace these with `android:layout_marginHorizontal`

File: android/java/res/layout/brave_wallet_add_network.xml [hunk 2]
```diff
@@ -219,20 +219,21 @@
...
+            android:layout_marginBottom="12dp"
             android:paddingTop="8dp"
             android:paddingBottom="8dp"
-            android:paddingStart="16dp"
-            android:paddingEnd="16dp"
+            android:paddingStart="8dp"
+            android:paddingEnd="8dp"
```
Comment (Pavneet-Sing) [inline android/java/res/layout/brave_wallet_add_network.xml:236]: nit: can be replaced with [android:paddingHorizontal](https://developer.android.com/reference/android/view/View#attr_android:paddingHorizontal)

**Review by Pavneet-Sing — APPROVED** [review]
++

**Comment by Pavneet-Sing** [comment]
The corresponding issues has been resolved by the [#13069](https://github.com/brave/brave-core/pull/13069) PR.

**Pull request closed without merging**